data/login_data.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
here, instead of using json module you can use python module : shelve, it creates a file which contains every type of python objects(classes, funtion, instances of class, dictionary etc), which are held in dictionary. Example of using:
first file:
dict = {'a':1, 'b':2, 'c':3}
import shelve
a = shelve.open('new')
a['passwords'] = dict
a.close()

second file:
import shelve
a = shelve.open('new')
print(a['passwords'])

Python creates file itself!!